lessons/pydata/webscraping/index.ipynb

```diff
@@ -0,0 +1,6844 @@
+{
```

Comment:
<p>Dalo by se tohle přesunout někam do appendixu s tím, že to na hodině nebudou účastnice prakticky dělat? Klidně to ukaž, ale vážně bych se bál, že trivialita typu nainstalujte si Ghostscript může hodinu narušit neúměrně praktickému významu. Btw. o tomhle by byl super lightning talk na PyData :-D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Jo...to je dobrý nápad. Zrovna si rvu vlasy, že to nemám kde vyzkoušet pod windows.</p>